genesyscloud/platform/platform.go

```diff
@@ -123,7 +123,7 @@ func init() {
 	path, err := detectExecutingBinary()
 	if err != nil {
 		log.Printf(`Error detecting binary: %v`, err)
-		platformConfigSingleton.platform = PlatformUnknown
+		platformConfigSingleton.platform = PlatformTerraform // use default terraform binary. so it wont break regression
```

Comment:
This is a sanity check to ensure that a valid platform is in use. I agree we need to default to the Terraform platform binary, which we can do in `Binary()`